plugins/test/opengrok/auth/plugin/util/DummyHttpServletRequestLdap.java

```diff
@@ -83,8 +90,9 @@ public int getMaxInactiveInterval() {
             return 3600;
         }
 
+        @Override
         @SuppressWarnings("deprecation")
```

Comment:
and so can you remove the deprecation?

Reply:
No, the method is still deprecated. Both method and the `HttpServletContext` class are deprecated. Maybe adding `@Deprecated` will solve the issue? However, not sure if one or the other is better.